docs/src/app/components/AppLeftNav.jsx

```diff
@@ -81,6 +114,23 @@ const AppLeftNav = React.createClass({
         <div style={prepareStyles(styles.logo)} onTouchTap={this.handleTouchTapHeader}>
           Material-UI
         </div>
+
+        <DropDownMenu
```

Comment:
What about something like this instead:
![capture d ecran 2016-02-19 a 21 21 01](https://cloud.githubusercontent.com/assets/3165635/13187948/b4bf623a-d74e-11e5-8aa2-50b0c8d4843e.png)